wokspace-loader/src/index.ts

```diff
@@ -83,29 +96,22 @@ async function openIdeAfterRunning(ws: che.IWorkspace) {
     });
 
     if (!ideUrl) {
-        ideUrl = ws.links.ide;
+        ideUrl = workspace.links.ide;
     }
 
-    console.log('IDE url is: ', ideUrl);
-    document.location.assign(ideUrl);
+    console.log('IDE URL: ', ideUrl);
+    // document.location.assign(ideUrl);
```

Comment:
Loader will never redirect to Theia or GWT IDE, remove comment on this line

Reply:
Done